execution_engine2.spec

```diff
@@ -67,72 +67,58 @@
         /*  Running long running methods through Docker images of services from Registry  */
         /*================================================================================*/
         /* A workspace object reference of the form X/Y or X/Y/Z, where
-           X is the workspace name or id,
```

Comment:
UPAs are now required. Per @briehl, the Narrative supplies UPAs anyway

Reply:
Just to make sure, a full UPA is required now? numerical X/Y/Z? In line 69, it says X/Y or X/Y/Z are allowed.

Reply:
Good eye, removed the non-UPA format from the line.